.github/workflows/deployTest.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 concurrency:
   group: test-deploy
-  cancel-in-progress: true
+  cancel-in-progress: false
```

Comment:
I would guess so, but is setting false here the same as getting rid of it? We don't appear to have this in our other deploy files.